src/main/java/gameframework/drawing/GameUniverseViewPortDefaultImpl.java

```diff
@@ -26,7 +26,11 @@ public GameUniverseViewPortDefaultImpl(GameData data) {
 	}
 
 	protected URL backgroundImage() {
-		return backgroundImage("/images/background_image.gif");
+		return backgroundImage("../../images/black_background.png");
```

Comment:
background_image.gif just doesn't exist, so we created black_background.png and we wanted to put images in resources too with "images" folder.